frontend/src/pages/CalendarPage/index.js

```diff
@@ -1,11 +1,26 @@
 import Component from '@/utils/Component';
+import Week from '@/components/Week/index';
+import Calendar from '@/components/Calendar/index';
+import TotalSum from '../../components/TotalSum';
+import './index.scss';
 
 export default class CalendarPage extends Component {
   template() {
     return `
-      <div>
-        <h1>Calendar Page</h1>
+      <div class = "calendarPage">
+        <div class="week">
```

Comment:
```suggestion
      <div class="calendarPage">
        <div class="week">
```
class 는 띄어쓰기를쓰지 않고 붙여서 쓰는게 좋아 보입니다!

Reply:
아하, 요 부분은 제가 자잘하게 나오는 나쁜 코딩 습관인 것 같습니다. 마크업 부분들 보고 띄어쓰기 있는 것도 수정해두겠습니다!